packages/forma-36-website/src/components/Layout.css

```diff
@@ -1,26 +1,17 @@
 @import '~@contentful/f36-tokens/dist/css/index.css';
 
-* {
-  box-sizing: border-box;
-}
-
 body {
-  font-family: var(--font-stack-primary);
-  font-weight: var(--font-weight-normal);
-  font-size: 16px;
-  line-height: var(--line-height-default);
-  margin: 0;
   min-width: 1280px;
 }
 
 pre {
   margin: 0;
   white-space: normal;
-  font-family: var(--font-stack-primary);
+  font-family: var(--font-stack-primary) !important;
 }
 
 pre.prism-code {
-  font-family: var(--font-stack-monospace);
+  font-family: var(--font-stack-monospace) !important;
```

Comment:
Why `!important`?

Reply:
I believe that the prism library we use for highlighting the code already has a font-family set. And due to specificity it needs the !important flag, to correct override that.

Reply:
got it